src/compose/ComposeBox.js

```diff
@@ -128,7 +129,10 @@ class ComposeBox extends PureComponent {
         <AutoCompleteView text={message} onAutocomplete={this.handleAutoComplete} />
         <View style={[styles.composeBox, { height: totalHeight }]}>
           <View style={componentStyles.bottom}>
-            <ComposeMenu />
+            <ComposeMenu
+              handleMessageChange={this.handleMessageChange}
```

Comment:
Yep, actually, this will go to `master` soon.
I did implement it in https://github.com/zulip/zulip-mobile/pull/996